### PR TITLE
DO_NOT_MERGE template service broker: remove requester username from catalog schema

### PR DESCRIPTION
**Pull request by jim-minter**
builds upon https://github.com/openshift/origin/pull/14586 and removes the requester username from catalog schema (the parameter it is still needed on provision and bind calls)

### CODE REVIEW

File: pkg/template/servicebroker/catalog.go [hunk 1]
```diff
@@ -14,16 +14,6 @@ import (
 	templateapi "github.com/openshift/origin/pkg/template/api"
 )
 
-const (
-	namespaceTitle       = "Template service broker: namespace"
-	namespaceDescription = "OpenShift namespace in which to provision service"
```
Comment (bparees) [inline pkg/template/servicebroker/catalog.go]: we'll still want this change though.

**Comment by bparees** [comment]
@jim-minter after talking with @jwforres we're going to have the TSB keep sending the username parameter, but the console will hide it and autopopulate it.  That was preferred over having the console try to know when to/not to inject the parameter.

So this PR can be closed, with the exception of the constants cleanup you were doing for the namespace parameter (which i guess technically should have been part of other commit, but oh well)

**Comment by jim-minter** [comment]
that's cool - I've slipped the cleanups into #14634.

**Pull request closed without merging**